docs/content/preview/yugabyte-voyager/migrate/live-fall-forward.md

```diff
@@ -507,86 +529,100 @@ Perform the following steps as part of the cutover process:
+1. Quiesce your target YugabyteDB database, that is stop application writes.
+1. Perform a cutover after the exported events rate ("Export rate" in the metrics table) drops to using the following command:
 
     ```sh
-    yb-voyager fall-forward switchover --export-dir <EXPORT_DIR>
+    # Replace the argument values with those applicable for your migration.
+    yb-voyager initiate cutover to source-replica --export-dir <EXPORT_DIR>
     ```
 
-    Refer to [fall-forward switchover](../../reference/fall-forward/fall-forward-switchover/) for details about the arguments.
+    Refer to [cutover to source-replica](../../reference/cutover-archive/cutover/#cutover-to-source-replica) for details about the arguments.
 
-    The `fall-forward switchover` command stops the `fall-forward synchronize` process, followed by the `fall-forward setup` process after it has imported all the events to the fall-forward database.
+    The `initiate cutover to source-replica` command stops the `export data from target` process, followed by the `import data to source-replica` process after it has imported all the events to the source-replica database.
 
-1. Wait for the switchover process to complete. Monitor the status of the switchover process using the following command:
+1. Wait for the cutover process to complete. Monitor the status of the cutover process using the following command:
 
     ```sh
-    yb-voyager fall-forward status --export-dir <EXPORT_DIR>
+    # Replace the argument values with those applicable for your migration.
+    yb-voyager cutover status --export-dir <EXPORT_DIR>
     ```
 
-    Refer to [fall-forward status](../../reference/fall-forward/fall-forward-switchover/#fall-forward-status) for details about the arguments.
+    Refer to [cutover status](../../reference/cutover-archive/cutover/#cutover-status) for details about the arguments.
 
-1. Setup indexes and triggers to the fall-forward database manually. Also, re-enable the foreign key and check constraints.
+1. Setup indexes and triggers to the source-replica database manually. Also, re-enable the foreign key and check constraints.
 
-1. Verify your migration. After the schema and data import is complete, the automated part of the database migration process is considered complete. You should manually run validation queries on both the source and fall-forward databases to ensure that the data is correctly migrated. A sample query to validate the databases can include checking the row count of each table.
+1. Verify your migration. After the schema and data import is complete, the automated part of the database migration process is considered complete. You should manually run validation queries on both the source and source-replica databases to ensure that the data is correctly migrated. A sample query to validate the databases can include checking the row count of each table.
```

Comment:
source and source replica? not target and source-replica?

Reply:
yes correct target and source-replica